src/index.js

```diff
@@ -205,7 +193,7 @@ export default function undoable (reducer, rawConfig = {}) {
     initialState: rawConfig.initialState,
     initTypes: parseActions(rawConfig.initTypes, ['@@redux/INIT', '@@INIT']),
     limit: rawConfig.limit,
-    filter: rawConfig.filter || () => true,
+    filter: rawConfig.filter || (() => true),
```

Comment:
Good thing I left them in, then :sweat_smile: It doesn't really hurt anything either (still really readable), so let's just leave it like that. :wink: